CMakeLists.txt

```diff
@@ -80,6 +80,6 @@ set_property(TARGET pprepair PROPERTY CXX_STANDARD 17)
 add_to_cached_list( CGAL_EXECUTABLE_TARGETS pprepair )
 
 # Link the executable to CGAL and third-party libraries
-target_link_libraries(pprepair ${GDAL_LIBRARY} ${Boost_LIBRARIES})
+target_link_libraries(pprepair ${GDAL_LIBRARY} ${Boost_LIBRARIES} stdc++fs)
```

Comment:
According to https://en.cppreference.com/w/cpp/filesystem#Notes, this is probably called `c++fs` under `libc++`. Might be worth doing something like:

```cmake
if (CMAKE_COMPILER_IS_GNUCC AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 9.1)
  target_link_libraries(pprepair stdc++fs)
endif()
```

(and similar for Clang < 9.0)

Reply:
> Might be worth doing something like:

I must admit that I am not familiar with the different compilers. I trust your experience.

An alternative would be to use `boost::filesystem`. I just tried it and it seems to work. `boost` is needed anyway?!

Reply:
I think it's a good idea to use the standard library. But why does it even link to Boost, it's not used.

Reply:
_Boost_ is needed by _cgal_. I've removed `stdc++fs`. Thanks for the hint!

Reply:
But you said it doesn't build on Debian without `stdc++fs`?

Reply:
Small update on this, after testing with `CXX=clang++ CXXFLAGS=-stdlib=libc++` and `stdc++fs` in the library list, it appears that in this configuration, Clang will try to link with the libstdc++ `libstdc++fs` (a static library on my distro, Fedora 39), which sounds like a bad idea. I didn't check whether the final binary crashes, but it's better not to mix them.